namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/pingdom.tf

```diff
@@ -0,0 +1,17 @@
+
+resource "pingdom_check" "cloud-platform-prometheus-live-0-healthcheck" {
```

Comment:
Hi. Can you change `cloud-platform-prometheus-live-0-healthcheck`, it shouldn't really make a difference other than readability.

Reply:
done @jasonBirchall